helm-charts/daskhub/values.yaml

```diff
@@ -157,20 +157,44 @@ dask-gateway:
     extraConfig:
       optionHandler: |
         from dask_gateway_server.options import Options, Integer, Float, String, Mapping
+        import string
+
+        # Escape a string to be dns-safe in the same way that KubeSpawner does it.
+        # Reference https://github.com/jupyterhub/kubespawner/blob/main/kubespawner/spawner.py#L1828-L1835
```

Comment:
```suggestion
        # Reference https://github.com/jupyterhub/kubespawner/blob/616f72c4aee26c3d2127c6af6086ec50d6cda383/kubespawner/spawner.py#L1828-L1835
```

I made this a permalink as otherwise this comment may point to the wrong lines over time.

![image](https://user-images.githubusercontent.com/3837114/170261802-07cf31bc-9b4f-46ac-8a5a-816db3ccdf7d.png)

Reply:
Perfect! Thank you! ✨